numpy/core/fromnumeric.py

```diff
@@ -908,11 +908,12 @@ def sort(a, axis=-1, kind=None, order=None):
 
     .. versionadded:: 1.12.0
 
-    quicksort has been changed to an introsort which will switch
-    heapsort when it does not make enough progress. This makes its
-    worst case O(n*log(n)).
+    quicksort has been changed to an introsort algorithm. 
```

Comment:
"changed to introsort, which ..." is preferable. It would be useful to add a link, maybe to Wikipedia.